be/src/vec/exec/volap_scan_node.cpp

```diff
@@ -554,8 +576,11 @@ Block* VOlapScanNode::_alloc_block(bool& get_free_block) {
 int VOlapScanNode::_start_scanner_thread_task(RuntimeState* state, int block_per_scanner) {
     std::list<VOlapScanner*> olap_scanners;
     int assigned_thread_num = _running_thread;
-    size_t max_thread = std::min(_volap_scanners.size(),
-                                 static_cast<size_t>(config::doris_scanner_thread_pool_thread_num));
+    size_t max_thread = config::doris_scanner_queue_size;
```

Comment:
If this code change is not sure to be effective, I think it can not be modified first

Reply:
The previous logic is wrong, For example, if assiged thread num = 13 and volap_scanner.size = 5.
thread_slot_num = std::min(thread_slot_num, max_thread - assigned_thread_num);
                if (thread_slot_num <= 0) {
                    thread_slot_num = 1;
                }
This logic will set thread_slot_num = 1. Will not put as many as task to queue.